content/en/docs/tasks/security/authn-policy/index.md

```diff
@@ -19,6 +19,12 @@ the underlying concepts in the [authentication overview](/docs/concepts/security
 [installation steps](/docs/setup/install/kubernetes/#installation-steps), or set `global.mtls.enabled` to false using
 [Helm](/docs/setup/install/helm/)).
 
+{{< warning >}}
```

Comment:
Consider changing the text of the warning to:
```
If the authentication policy enables mutual TLS on a proxy that receives requests from different trust domains, turn off the {{< gloss >}}trust domain validation{{< /gloss >}}. Otherwise, the validation rejects the request.
```